### PR TITLE
Added the Hello World article for Tex.

### PR DESCRIPTION
**Pull request by Bracciata**
Fixes issue #208

### CODE REVIEW

File: projects/hello-world/_posts/2020-09-30-tex.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Hello World in Tex
+layout: default
+last-modified: 2020-09-30
+tags: [tex, hello-world]
+authors:
+  - bracciata
```
Comment (jrg94) [inline projects/hello-world/_posts/2020-09-30-tex.md:7]: Make sure your name is in the authors file: https://github.com/TheRenegadeCoder/sample-programs-website/blob/master/_data/authors.yml

Comment (Bracciata) [reply projects/hello-world/_posts/2020-09-30-tex.md:7]: Should I do my full name or just my handle?

Comment (Bracciata) [reply projects/hello-world/_posts/2020-09-30-tex.md:7]: On it!

Comment (Bracciata) [reply projects/hello-world/_posts/2020-09-30-tex.md:7]: Made a new pull request doing so #321

**Review by jrg94 — APPROVED** [review]
This is great stuff! Usually, I'd ask you to write a little about how to run—or in this case render—the solution, but we're really swamped for Hacktoberfest. If you want to modify this in the future, go for it! Otherwise, it might be a good idea to add an issue for it.

**Pull request merged**